core-bundle/tests/Security/TwoFactor/AuthenticatorTest.php

```diff
@@ -126,4 +126,17 @@ public function testGeneratesTheQrCode(): void
         $this->assertSame(5897, \strlen($svg));
         $this->assertSame(0, strpos($svg, $beginSvg));
     }
+
+    private function randomBytes(int $bytesCount, int $seed): string
```

Comment:
Can we rename this? I know I'm being picky but it's not "random" anymore now. And it doesn't have to be. Let's just name it `generateSecret()` and add a comment about it having to be deterministic for tests or so. Otherwise someone will read this code and go like "wtf?".

Reply:
Done in c10dc43c55f7e49aeaf7a8c13deb10bd5d308482